internal/dev_server/ui/src/IconProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from "react";
```

Comment:
This is copy-paste: add the icon sprite from `@launchpad-ui/icons` so that the icons can be sourced properly